doc/user-guide/github.md

```diff
@@ -69,4 +76,28 @@ Without `--watch`, the logs will look like this:
 
 ![](../static/github/logs.png)
 
-A complete sample project is [available here.](https://github.com/edublancas/cloud-template)
\ No newline at end of file
+A complete sample project is [available here.](https://github.com/edublancas/cloud-template)
+
+## Configure action through CLI
+
+If your project is already hosted on Github, running `ploomber-cloud init` inside your repository folder will initialize
+the project and prompt for configuring Github action:
+
+```bash
+Your app 'snowy-disk-4284' has been configured successfully!
+Do you want to configure a github action? [y/N]: 
+```
+
+On confirming with `y` the CLI will create a `ploomber-cloud.yaml` file in the path `.github/workflows`.
+In order to trigger an action for deploying the project using Github actions you need to add, commit and push this file along with the `ploomber-cloud.json`.
+
+Once done, you can monitor progress as discussed [above](monitor). Ensure that the API key is set as Github secret.
+
+In case the [workflow template](https://github.com/edublancas/cloud-template/blob/main/.github/workflows/ploomber-cloud.yaml) has been updated, and you need to re-initialise the application the CLI will prompt your for updating the workflow file:
```

Comment:
let's keep spelling consistent:

re-initialise -> re-initialize
github -> GitHub

Reply:
Have added more mentions of `GitHub` in the latest commit.